src/backend/distributed/planner/recursive_planning.c

```diff
@@ -73,19 +75,6 @@
 #include "utils/guc.h"
 
 
-/*
- * RecursivePlanningContext is used to recursively plan subqueries
- * and CTEs, pull results to the coordinator, and push it back into
- * the workers.
- */
-typedef struct RecursivePlanningContext
-{
-	int level;
-	uint64 planId;
-	List *subPlanList;
-	PlannerRestrictionContext *plannerRestrictionContext;
-} RecursivePlanningContext;
```

Comment:
Maybe just define a wrapper for `RecursivelyPlanSubqueriesAndCTEs` so we can keep this here.

Reply:
Ah, I thought you asked for `RecursivePlanningContext ` wrapper. Sure, it makes sense to have a wrapper for `RecursivePlanningContext `